doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -202,8 +202,8 @@ An over use of host aggregates and availability zones can result in a granular p
 Recommendation: Separation of control zone and execution zone into different security zones
 
 <a name="3.3.4"></a>
-### 3.3.4. Flavour management (i.e. T-Shirt type)
-Content to be developed
+### 3.3.4. Flavor management
+A flavor defines the compute, memory, and storage capacity of nova instances. When instances are spawned, they are mapped to flavors which define the available hardware configuration for them. For simplicity, the flavors can be named according to T-shirt sizes like XS,S,M,L,XL,2XL,3XL and so on. The specifications for these sizes can be defined based on the  category of workloads in terms of resource requirements.
 
```

Comment:
The text doesn't include network capabilities in the flavor. Is it intentionally?

Reply:
A) Are there any particular capabilities you want to be highlighted ? 
OR 
B) We should state a few examples in this section and add a hyperlink to OpenStack documentation saying that ```For advanced flavor configuration, refer URL ...``` as being discussed in https://github.com/cntt-n/CNTT/pull/661#issuecomment-557009320.

Reply:
In the end we need to map between RM chapter 4 and OpenStack. However RM doesn't provide clear guidance yet. At least I think we should know what to do for Network Intensive profile in difference to Basic and Compute Intensive.

Reply:
we can refer to RM flavours: https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter04.md#4211-predefined-compute-flavours

Reply:
Ok, I have given a reference to this section.